### PR TITLE
[Term Entry] for Kotlin date time API: .plus()

### PR DESCRIPTION
**Pull request by telfort92**
### Description

- Added the datetime.plus() term for kotlin
- Closes Issue #3818 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/datetime/terms/plus/plus.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+Title: 'Plus() method'
```
Comment (avdhoottt) [inline content/kotlin/concepts/datetime/terms/plus/plus.md:2]:
```suggestion
Title: '.plus()' 
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@telfort92 This entry does not follow the style guide. Please read this [documentation](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) and format your content according to it.

**Comment by telfort92** [comment]
ok I will make the necessary changes

**Comment by avdhoottt** [comment]
@telfort92 You have used the syntax and examples of the .plus method that refers to the Java libraries. Please replace this with the correct syntax/examples from the kotlix.datetime library.

**Pull request closed without merging**